scalafix-interfaces/src/main/java/scalafix/interfaces/ScalafixArguments.java

```diff
@@ -110,6 +110,11 @@ ScalafixArguments withToolClasspath(
      */
     ScalafixArguments withMode(ScalafixMainMode mode);
 
+    /**
+     * @param dialect The dialect to use in order to parse source code. Can be aither "Scala2" or "Scala3"
```

Comment:
No need to repeat what the enum says - this might not age well otherwise
```suggestion
     * @param dialect The dialect to use in order to parse source code.
```